tests/docker-compose.yml

```diff
@@ -1,26 +1,35 @@
 version: "3"
 services:
-  centos6:
-    build: ./dockerfiles/centos6
-    cap_add:
-      - ALL
+
+  centos8:
+    build: ./dockerfiles/centos8
+    cap_add: [ALL]
   centos7:
     build: ./dockerfiles/centos7
-    cap_add:
-      - ALL
-  debian8:
-    build: ./dockerfiles/debian8
-    cap_add:
-      - ALL
+    cap_add: [ALL]
+  centos6:
+    build: ./dockerfiles/centos6
+    cap_add: [ALL]
+
+  debian10:
```

Comment:
Same here, keep ASC version ordering

Reply:
up to you, but please my comment below, and "double confirm" your decision: ASC (or DESC).

Thank you

Reply:
Done in 6df0663b4097bb79cdced899c52e578f74ec0c00. Please confirm it 😉

Reply:
@pkuczynski we keep the Ascending order, right?